app/views/alaveteli_pro/info_requests/new.html.erb

```diff
@@ -62,16 +62,15 @@
         <%= fields_for @outgoing_message do |o| %>
           <p>
             <label class="form_label" for="outgoing_message_body">
-              <%= _('Your request:') %></label>
+              <%= _('Your request') %></label>
             <%= o.text_area :body, :rows => 20, :cols => 60 %>
           </p>
         <% end %>
 
         <div class="form_button">
-          <button><%= _("Save draft") %></button>
```

Comment:
Sorry, it's not explained but these need to be buttons so that they both submit the form, but we can pass a name and value with the preview one (it doesn't work if they're submit input unfortunately). Can we style the <button> tags the same here, or is that breaking some other semantics?